### PR TITLE
Data/Globals: Fix - Commands: 'disable, enable'.

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: tests/qunit/core/editor/container/container.spec.js [hunk 1]
```diff
@@ -1,27 +1,39 @@
...
-						type: 'fake',
-						id: 'fake',
-						settings: new Backbone.Model(),
-						model: new Backbone.Model(),
-						label: 'Fake Label',
-					},
-					container = new elementorModules.editor.Container( fakeArgs );
+		QUnit.test( 'constructor()', ( assert ) => {
+			const fakeArgs = {
+					type: 'fake',
```
Comment (matipojo) [inline tests/qunit/core/editor/container/container.spec.js:7]: Always prefer `test` rather than `fake`.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: tests/qunit/core/editor/container/container.spec.js (hunk 1 above)
Comment (iNewLegend) [reply tests/qunit/core/editor/container/container.spec.js:7]: Its old tests, are you sure add it here?

Comment (matipojo) [reply tests/qunit/core/editor/container/container.spec.js:7]: Opps, ignore it for here, but for the next tests...

**Review by matipojo — APPROVED** [review]

**Comment by matipojo** [comment]
Send it to Elementor & add the dependency in the description

**Pull request closed without merging**